pom.xml

```diff
@@ -413,8 +413,15 @@
 				<groupId>org.openmrs.module</groupId>
 				<artifactId>webservices.rest-omod-common</artifactId>
 				<version>${webservices.restModuleVersion}</version>
-				<scope>test</scope>
+				<scope>provided</scope>
 			</dependency>
+			<dependency>
```

Comment:
@reagan-meant Can you format this section to be consistent with the rest, indentation and a space before the dependency

Reply:
Done